src/patterns/definitions.owl

```diff
@@ -50828,797 +50679,27 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2060107> "morphology of lumbar vertebra"^^xsd:string)
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/OBA_2060107> "lumbar vertebra morphology"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2060107> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0002414>)))
-
-# Class: <http://purl.obolibrary.org/obo/OBA_2060108> (lung morphology)
-
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2060108> "The morphology of a lung."^^xsd:string)
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2060108> "morphology of lung"^^xsd:string)
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/OBA_2060108> "lung morphology"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2060108> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0002048>)))
-
-# Class: <http://purl.obolibrary.org/obo/OBA_2060109> (male reproductive organ morphology)
-
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2060109> "The morphology of a male reproductive organ."^^xsd:string)
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2060109> "morphology of male reproductive organ"^^xsd:string)
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/OBA_2060109> "male reproductive organ morphology"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2060109> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0003135>)))
-
-# Class: <http://purl.obolibrary.org/obo/OBA_2060110> (mammillothalamic tract of hypothalamus morphology)
```

Comment:
Why are there more than 1000 lost classes here? This seems wrong!

Reply:
I did remove 118 recently added terms from `entity_attribute.tsv` whose logical axioms look like duplications of terms that already exist in `entity_attribute_part_of.tsv` [in another PR.](https://github.com/obophenotype/bio-attribute-ontology/pull/202/files)

* For example:
    1. existing term in `entity_attribute_part_of.tsv`:
```
1956:OBA:0005653                UBERON:0002990  mammillothalamic tract of hypothalamus  PATO:0000051    morphology
```
    2. intended new term in `entity_attribute.tsv`:

```
4213: OBA:2060110		UBERON:0002990	mammillothalamic tract of hypothalamus	PATO:0000051	morphology
```

I will do some more investigation to figure out of the rest making up the thousand lost classes.

Reply:
I have checked the diff of `definitions.owl`, and the 255 removed classes are all intended to be replaced  in the present PR, obsolete GO terms, or had been already dropped in PR #202 .

Reply:
This does not sound right - IDs should never be deleted. At most, we can delete the 120 or so you discovered as duplicates, but not the 255 - these should not have changed their IDs. Even if you move the terms to a new pattern, the ID should be the same, unless I am misunderstanding something here.